templates/item/item-ability-sheet.hbs

```diff
@@ -52,35 +63,27 @@
                                 placeholder=""
                             />
                         </div>
-                    </div>
-                    <div class="hgss-box align-items-center">
-                        <div class="hgss-header mb-1" style="padding: 6px 5px;">
-                            <label for="system.frequency">Frequency:</label>
-                        </div>
-                        <div class="hgss-body">
-                            <input
-                                class="ml-2"
-                                name="system.frequency"
-                                type="text"
-                                value="{{system.frequency}}"
-                                placeholder=""
-                            />
-                        </div>
```

Comment:
Why is this removed? Frequency exists in abilities (f.e. Intimidate)

Reply:
this shouldn't be removed i'll add it back in. not sure why it is

Reply:
added this back in